<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalAndDBinfoLine</class>
 <widget class="QWidget" name="LocalAndDBinfoLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>46</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLineEdit" name="localValue"/>
   </item>
   <item>
    <widget class="QLabel" name="fieldName">
     <property name="text">
      <string>field</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="removeValue"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
